playwright/tests/page-objects/overview.ts

```diff
@@ -23,7 +24,8 @@ export class Overview {
     this.titleSection = this.overview.locator('.overview-title-section');
     this.title = this.titleSection.locator('.overview-title');
     this.createButton = this.titleSection.getByRole('button').last();
-    this.search = this.overview.locator('input');
+    this.sorting = this.overview.locator('input');
+    this.search = this.overview.locator('Sort By');
```

Comment:
seems to be wrong and you don't use the sorting locator anywhere, so I would just remove it again